.gitlab-ci.yml

```diff
@@ -0,0 +1,14 @@
+include: https://gitlab-templates.ddbuild.io/compute-delivery/v2/compute-delivery.yml
+
+test:
+  stage: verify
+  tags: [ "arch:amd64" ]
+  image: registry.ddbuild.io/images/mirror/golang:1.21.5
```

Comment:
> [announce.dev] [ACTION REQUIRED] Please upgrade Go to v1.22 by July 31, 2024

Should this be part of the equation here ?

Reply:
Damn, thanks!